.github/workflows/container-image.yaml

```diff
@@ -41,17 +41,13 @@ jobs:
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v2
 
-      - name: Docker meta
-        id: meta
-        uses: docker/metadata-action@v4
-        with:
-          images: |
-            ghcr.io/${{ github.repository }}
-          tags: |
-            type=ref,event=branch
-            type=ref,event=pr
-            type=semver,pattern={{version}}
-            type=semver,pattern={{major}}
+      - name: Inspect builder
```

Comment:
Curious about this change, is it needed?

Reply:
@sts ?

I think this is mostly informational.

Reply:
yes, this is just to include information about the build in the ci output.